synapse_invite_checker/invite_checker.py

```diff
@@ -1132,30 +1151,26 @@ async def room_scan(self) -> None:
 
                 # only shut down rooms that only have EPA hosts in them
                 if await self.have_all_pro_hosts_left(room_id):
-                    last_user_left_timestamp: int | None = (
-                        await self.get_timestamp_from_eligible_events_for_epa_room_purge(
-                            room_id
-                        )
+                    epa_room_result = await self.define_criteria_to_kick_users_epa_mode(
```

Comment:
the old name was probably better, apart from that it was timestamp instead of timestamps.